practical-07/src/components/CarDetails.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+<div class="containerCard">
+    <router-link to="/">
+      <button class="back-btn" size="sm" pill variant="outline">
+          <i class="fa fa-arrow-circle-left btn-back"></i>
+          <span class="separator"/>
+          <span class="text">Back</span>
+      </button>
+    </router-link>
+    <div class="containerDetails">
+      <div class="carName">{{ carInfo.name }}</div>
+      <img :src="carInfo.image" alt="car-image" />
+      <div class="carDetails">{{ carInfo.details }}</div>
+      <div><b>Price:</b>{{ carInfo.price }}</div>
+    </div>
+</div>
+</template>
+
+<script>
+import { mapState, mapActions } from "vuex";
+export default {
+  name: 'CarDetails',
+  created() {
+    this.getCarDetails(this.$route.params.id);
+  },
+
+  data() {
+    return {
+      
+    };
+  },
+  computed: {
+    ...mapState({
+      carInfo: (state) => state.carDetails,
+    }),
+  },
+
+  methods: {
+    ...mapActions({
+      getCarDetails: "getCarInfo",
+    }),
+  },     
+};
+</script>
+
+<style>
+.containerCard {
+  background-color: rgb(187, 234, 235);
+  border-radius: 15px;
+  margin: 25px;
+  box-shadow: rgba(112, 112, 220, 0.1) 0px 8px 24px, rgba(223, 28, 28, 0.1) 0px 16px 56px, #f265651a 0px 24px 80px;
```

Comment:
Codacy has a fix for the issue: [Insert `⏎···`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
  box-shadow: rgba(112, 112, 220, 0.1) 0px 8px 24px,
```